keps/sig-apps/2879-ready-pods-job-status/README.md

```diff
@@ -39,16 +39,16 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 - [x] (R) KEP approvers have approved the KEP status as `implementable`
 - [x] (R) Design details are appropriately documented
 - [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
-  - [ ] e2e Tests for all Beta API Operations (endpoints)
-  - [ ] (R) Ensure GA e2e tests for meet requirements for [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+  - [x] e2e Tests for all Beta API Operations (endpoints)
+  - [ ] (R) Ensure GA e2e tests for meet requirements for [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md)
```

Comment:
I don't see any e2e tests specific to this feature in https://github.com/kubernetes/kubernetes/blob/master/test/e2e/apps/job.go. However, the test plan enlists the Beta graduation assumes the tests are passing: https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2879-ready-pods-job-status#beta. I'm wondering if we should add some e2e tests. WDYT @alculquicondor ?

Reply:
I don't think this is necessary. E2E tests are more important when there are new endpoints or for more complex features.

Reply:
Looking at the E2E tests, it is surprising that we don't verify status at all.

Reply:
Yes, there should be test ensuring the implementation, although I don't ever expect this test to be conformance, since as mentioned it will test implementation details. But I feel more confident having a test which exercises this functionality explicitly.

Reply:
Added an e2e test to the test plan and graduation criteria